chapters/5-processing.tex

```diff
@@ -42,265 +46,354 @@
-household-level variables will be stored in one tidy dataset, 
-and household-member variables are reshaped to member level and stored in another tidy dataset,
+household-level variables will be stored in one tidy data table,
+and household-member variables are reshaped to member level
+and stored in another tidy data table,
 which also contains the household ID for each individual.
-So the household ID is intentionally duplicated in the household members table.
+The household ID is intentionally duplicated in the household members data table
+to allow one or several household members to be linked to the same household data.
+The household members data will be uniquely identified by
+either a single household member ID or
+a combination of household ID and household member ID.
 In the transaction data example,
-the result of the tidying process would be one transaction-level dataset, 
+the result of the tidying process would be one transaction-level data table,
 containing variables indicating the ID of all firms involved;
-and one firm-level dataset with a single entry for each firm.
-
-Reshaping tables is the most intricate task in this process,
-so you should be very familiar with commands such as \texttt{reshape} in Stata and \texttt{pivot} in R.
-Another key aspect in this process is to make sure that ID variables are consistent across tables,
-so they can always be linked.
-There must be a clear way to connect each tidy table to a master dataset.
-Note that the tidying process gets more complex as the number of nested groups increases.
+and one firm-level data table with a single entry for each firm.
+Then, firm-level analysis is easily done
+by calculating appropriate statistics in the transactions data table
+(in Stata, often through \texttt{collapse})
+and then merging or joining those results to the firms data table.
+
+In both examples above, your dataset is made up of two data tables,
+each saved in a separate file.
+In a tidy workflow, your dataset is a set of one or more data tables,
+where each data table is a file in your data storage.
+There must also be a clear way to connect each
+tidy data table to a master dataset,
+and thereby also to all other datasets.
+This is done by deciding which data table is the main data table,
+and that data table's unit of observation is
+the main unit of observation of your dataset.
+The main unit of observation must have a master dataset,
+and be listed in the data linkage table.
+All other data tables in your dataset must have
+an unambiguous way to merge with the main data table.
+This way, it will be possible to link
+all data points in all your project's datasets to each other.
+We recommend that you save your datasets as a folder,
+where the main data table share the same name as the folder,
+and the name of all other data tables start with the same name,
+but are suffixed with the unit of observation for that data table.
+
+In the household dataset example, 
```

Comment:
i think it would be helpful to describe in this paragraph of how the folder and files would be named, at least for one of the examples.

Reply:
I will do.

Reply:
@mariaruth , done in 28d259b787e504fd437b396b9845fc363481d738, please review